### PR TITLE
[CELEBORN-957] Simplify nano time duration calculation

### PR DESCRIPTION
**Pull request by cxzl25**
### What changes were proposed in this pull request?
use `TimeUnit.NANOSECONDS.toMillis` instead of `/1000_000`

### CODE REVIEW

**Review by zwangsheng — APPROVED** [review]
Thanks, LGTM

**Comment by zwangsheng** [comment]
BTW, maybe we can fix other place in this PR:

https://github.com/apache/incubator-celeborn/blob/bf0deae752bf79bd749c491571254b1a0798bbf3/common/src/main/scala/org/apache/celeborn/common/meta/WorkerInfo.scala#L239

File: common/src/main/scala/org/apache/celeborn/common/meta/WorkerInfo.scala [hunk 1]
```diff
@@ -236,7 +237,8 @@ class WorkerInfo(
        |ReplicatePort: $replicatePort
        |SlotsUsed: $slots
        |LastHeartbeat: $lastHeartbeat
-       |HeartbeatElapsedSeconds: ${(System.currentTimeMillis() - lastHeartbeat) / 1000}
+       |HeartbeatElapsedSeconds: ${TimeUnit.MICROSECONDS.toSeconds(
```
Comment (waitinfuture) [inline common/src/main/scala/org/apache/celeborn/common/meta/WorkerInfo.scala:240]: Should be `MILLISECONDS`?

Comment (cxzl25) [reply common/src/main/scala/org/apache/celeborn/common/meta/WorkerInfo.scala]: Thanks.

**Review by waitinfuture — APPROVED** [review]
LGTM, thanks! Merging to main/0.3

**Pull request closed without merging**